.gitattributes

```diff
@@ -38,68 +38,81 @@
 
-/config/gnu              ocaml-typo=prune
+asmcomp/*/emit.mlp       typo.tab=may typo.long-line=may
+asmcomp/power/NOTES.md   typo.missing-header
 
-emacs/*.el               ocaml-typo=long-line,unused-prop
-emacs/caml.el            ocaml-typo=long-line,unused-prop,missing-header
-emacs/COPYING            ocaml-typo=tab,non-printing,missing-header
-emacs/ocamltags.in       ocaml-typo=non-printing
+/config/gnu              typo.prune
 
-/experimental            ocaml-typo=prune
+emacs/*.el               typo.long-line=may
+emacs/caml.el            typo.long-line=may typo.missing-header
+emacs/COPYING            typo.tab typo.non-printing typo.missing-header
+emacs/ocamltags.in       typo.non-printing
 
-/manual                  ocaml-typo=prune
+/experimental            typo.prune
 
-ocamldoc/Changes.txt     ocaml-typo=missing-header
-ocamldoc/ocamldoc.sty    ocaml-typo=missing-header
-ocamldoc/odoc_index.html ocaml-typo=missing-header
+/manual                  typo.prune
 
-otherlibs/win32unix/readlink.c    ocaml-typo=long-line
-otherlibs/win32unix/stat.c        ocaml-typo=long-line
-otherlibs/win32unix/symlink.c     ocaml-typo=long-line
+ocamldoc/**              typo.long-line=may
+ocamldoc/Changes.txt     typo.missing-header
+ocamldoc/ocamldoc.sty    typo.missing-header
+ocamldoc/odoc_index.html typo.missing-header
 
-runtime/i386.S            ocaml-typo=long-line
+otherlibs/win32unix/readlink.c    typo.long-line
+otherlibs/win32unix/stat.c        typo.long-line
+otherlibs/win32unix/symlink.c     typo.long-line
 
-stdlib/hashbang     ocaml-typo=white-at-eol,missing-lf
+runtime/i386.S      typo.long-line
 
-testsuite/tests/**                                      ocaml-typo=missing-header
-testsuite/tests/lib-bigarray-2/bigarrf.f                ocaml-typo=missing-header,tab
-testsuite/tests/lib-unix/win-stat/fakeclock.c           ocaml-typo=
-testsuite/tests/misc-unsafe/almabench.ml                ocaml-typo=missing-header,long-line
-testsuite/tests/tool-toplevel/strings.ml                ocaml-typo=utf8,missing-header
-testsuite/tests/typing-unboxed-types/test.ml.reference* ocaml-typo=white-at-eof,missing-header
-testsuite/tests/win-unicode/*.ml                        ocaml-typo=utf8,missing-header
-testsuite/tools/*.S                                     ocaml-typo=missing-header
-testsuite/tools/*.asm                                   ocaml-typo=missing-header
-testsuite/typing                                        ocaml-typo=missing-header
+stdlib/hashbang     typo.white-at-eol typo.missing-lf
+
+testsuite/tests/**                                      typo.missing-header
+testsuite/tests/lib-bigarray-2/bigarrf.f                typo.tab
+testsuite/tests/lib-unix/win-stat/fakeclock.c           typo.missing-header=false
+testsuite/tests/misc-unsafe/almabench.ml                typo.long-line
+testsuite/tests/tool-toplevel/strings.ml                typo.utf8
+testsuite/tests/typing-unboxed-types/test.ml.reference* typo.white-at-eof
```

Comment:
```suggestion
```

Reply:
@dra27 did you miss the `*` at the end of the pattern?

Reply:
You're right, I had! In which case, it should be `typo.prune`, but perhaps this can be simplified by altering the tests. The following files:

```
tests/typing-unboxed-types/test.ml.reference-flat
tests/typing-unboxed-types/test.ml.reference-noflat
tests/translprim/array_spec.compilers.reference.flat
tests/translprim/module_coercion.compilers.reference.flat
tests/translprim/module_coercion.compilers.reference.no-flat
tests/translprim/array_spec.compilers.reference.no-flat
```
are the only ones which don't end `.reference`. Perhaps they should just be renamed to `.flat.reference` and `.no-flat.reference` so that the normal prune rule comes into play?